CONTRIBUTING.md

```diff
@@ -1,268 +1,4 @@
 <img src="assets/amber.png" alt="amber logo" width="250" align="right" />
 
 # Contributing to Amber
```

Comment:
GIven that there is an identical "Contributing" section in README.me, do we need CONTRIBUTING.md at all?  Consider removing this file.

Reply:
it is a github standard to have that file, we can think to remove it later maybe.

Reply:
Can we just make a link to that file in `README.md` instead?

Reply:
If we put a link to the readme I think that is enough to remove that file right now.